index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>How to write well-formed XHTML</title>
+  <meta charset="utf-8">
+</head>
+<body>
+<!-- Jordan Sutherland
+     assignment1.htm
+     INFO2340.WW
+     Thoendel
+     12/08/19
+-->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<script src="D:\jnsutherland\assignment01\script.js"></script>
```

Comment:
This is an absolute reference to your custom js on your computer. This means that it won't necessarily work on webserver or other computer when moved. A relative reference would be more appropriate and robust.